src/Sarif.FunctionalTests/SpecExamples/Recommended.sarif

```diff
@@ -0,0 +1,55 @@
+{
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "files": {
+        "file:///user/builder/work/src/collections/list.cpp": {
+          "mimeType": "text/x-c"
```

Comment:
This is actually correct, or at least it is "a" correct answer, because:

* [IANA](http://www.iana.org/assignments/media-types/media-types.xhtml) doesn't define a MIME type for C++.

* [freeformatter.com](http://www.freeformatter.com/mime-types-list.html#mime-types-list) (which gets its list from "the 'mime.types' file of the Apache HTTPD [virtual private server](http://www.webhostingsearch.com/virtual-private-server.php)") gives `x-c` for "C source file" but does not mention C++, but...

* A [stackoverflow page](http://stackoverflow.com/questions/1735659/list-of-all-mimetypes-on-the-planet-mapped-to-file-extensions) links to the [latest version](http://svn.apache.org/viewvc/httpd/httpd/trunk/docs/conf/mime.types?view=markup) of the Apache mime.types file, which says:

    ```
    text/x-c					c cc cxx cpp h hh dic
    ```